dojo/tools/awssecurityhub/compliance.py

```diff
@@ -14,6 +14,9 @@ def get_item(self, finding: dict, test):
         epss_score = None
         mitigation = finding.get("Remediation", {}).get("Recommendation", {}).get("Text", "")
         description = "This is a Security Hub Finding \n" + finding.get("Description", "")
+        description += f"\n**AWS Finding ARN:** {finding_id}\n"
+        description += f"AwsAccountId: {finding.get('AwsAccountId', '')}\n"
+        description += f"Region: {finding.get('Region', '')}\n"
```

Comment:
Could you please also add:
- GeneratorId --> Into Description
- ProductFields --> RecommendationUrl (see config_many_findings.json) --> "mitigation"

(Please go through all values and evaluate one by one which makes sense to also add).

Reply:
@manuel-sommer I've implemented these changes as you've suggested.  Mitigation was not in ProductFields for the Recommendation/Url.  I copied how L15 works and added to grab the Url in addition to the Text here.

Reply:
Actually, for recommendation URL for L16 should it be something like the below?  When I check in DefectDojo under 'Mitigation' field I don't see anything in that field.  There is also the possibility to maybe add references for the Url instead of adding into the Mitigation?  Though I think it would be better to include the Url as part of the mitigation since it is paired with the Recommendation Text already.

mitigation += f"Url: " + finding.get("Remediation", {}).get("Recommendation", {}).get("Url", "")

Does L15 need to be modified at all like 

mitigation = "Mitigation: \n" + finding.get("Remediation", {}).get("Recommendation", {}).get("Text", "")

Or are these two suggestions above not needed because the mitigation should properly join those values?  I checked in a Finding in DefectDojo and I do not see the Mitigation field being populated, but now checking this compliance.py L15 I'm curious if there's an issue there?

Reply:
@manuel-sommer what do you think?